contributed_definitions/NXoptical_system_em.nxdl.xml

```diff
@@ -10,4 +10,16 @@
     <field name="semi_convergence_angle" type="NX_NUMBER" units="NX_ANGLE"/>
     <field name="field_of_view" type="NX_NUMBER" units="NX_LENGTH"/>
     <field name="working_distance" type="NX_FLOAT" units="NX_LENGTH"/>
```

Comment:
definitions should be added to all fields

Reply:
done